Products/CMFPlone/PloneTool.py

```diff
@@ -617,7 +616,7 @@ def browserDefault(self, obj):
         # 7. If nothing else is found, fall back on the object's 'view' action.
         # 8. If this is not found, raise an AttributeError
 
-        # WebDAV in Zope is odd it takes the incoming verb eg: PROPFIND
+        # 0. WebDAV in Zope is odd it takes the incoming verb eg: PROPFIND
```

Comment:
should be ``9.``

Reply:
no, the list starts again, it the first action of above list

Reply:
Note the above list is the docstring as comment (b/c of security reasons as stated there). The number zero here is a real code comment about the following lines.